source/community.applications/usr/local/emhttp/plugins/community.applications/skins/Narrow/styles/community.applications.css

```diff
@@ -0,0 +1,2392 @@
+    cursor: default !important;
+  }
+  
+  body.stop-scrolling {
+    height: 70%;
+    overflow: inherit;  /* disable SweetAlert killing the scroll bar (stops the wiggle) */
+  }
+  
+  .body_sidebarScroll {
+    overflow: hidden !important;
+  }
+  
+  /* Typography */
+  a {
+    color: var(--a-color);
+  }
+  
+  .ca_bold {
+    font-weight: bold;
+  }
+  
+  .ca_italic {
+    font-style: italic;
+  }
+  
+  .ca_normal {
+    font-size: 1.4rem !important;
+  }
+  
+  .ca_note {
+    color: #d67777;
+  }
+  
+  .ca_red {
+    color: #882626;
+  }
+  
+  .ca_green {
+    color: #038503;
+  }
+  
+  /* Layout Components */
+  .ca_center {
+    margin: auto;
+    text-align: center;
+  }
+  
+  .ca_href {
+    cursor: pointer;
+  }
+  
+  /* Card and Container Styles */
+  .ca_holder,
+  .dockerHubHolder {
+    /* Layout */
+    display: inline-block;
+    float: left;
+    height: 20rem;
+    min-width: 24rem;
+    max-width: 24rem;
+    padding: 1.25rem 0.625rem 1.25rem 1.25rem;
+    margin: 0 1rem 1rem 0;
+    
+    /* Visual */
+    background-color: var(--template-background);
+    border: 1px solid var(--border-color);
+    overflow: hidden;
+    
+    /* Interactive */
+    cursor: pointer;
+  }
+  
+  .ca_holder:hover,
+  .dockerHubHolder:hover {
+    background-color: var(--template-hover-background);
+    box-shadow: 5px 5px 5px -3px rgba(255, 140, 47, 0.5);
+  }
+  
+  .ca_holderFav {
+    background-color: var(--template-favourite) !important;
+  }
+  
+  /* Home Templates Section */
+  .ca_homeTemplates {
+    display: flex;
+    flex-wrap: wrap;
+    height: 24.5rem;
+    overflow: hidden;
+  }
+  
+  .ca_homeTemplatesHeader {
+    font-size: 2rem;
+    margin-top: 1rem;
+    margin-bottom: 0.5rem;
+  }
+  
+  .ca_homeTemplatesLine2 {
+    font-size: 1.5rem;
+    margin-bottom: 1rem;
+  }
+  
+  /* Card Elements */
+  .ca_applicationName {
+    /* Layout */
+    width: 25rem;
+    max-height: 3rem;
+    margin-left: 0.75rem;
+    padding-top: 1.5rem;
+    
+    /* Typography */
+    font-size: 2rem;
+    font-weight: bold;
+    overflow: hidden;
+  }
+  
+  .ca_backgroundClickable {
+    height: 18.5rem;
+    cursor: pointer;
+  }
+  
+  .ca_bottomLine {
+    display: block;
+    position: relative;
+    top: 18rem;
+  }
+  
+  .ca_bottomLineSpotLight {
+    top: 15rem !important;
+  }
+  
+  .ca_categories {
+    font-size: 1rem;
+    font-style: italic;
+  }
+  
+  .cardCategory {
+    font-size: 1rem;
+    margin-left: 0.75rem;
+  }
+  
+  .cardDescription {
+    /* Layout */
+    display: block;
+    position: relative;
+    top: 0.5rem;
+    max-height: 7rem;
+    
+    /* Interactive */
+    cursor: pointer;
+    overflow: hidden;
+  }
+  
+  .cardDesc {
+    display: inline-block;
+    max-height: 6rem;
+    overflow: hidden;
+  }
+  
+  .cardWarning {
+    font-size: 1.5rem;
+    color: #bdbd14;
+  }
+  
+  /* Popup Button Styles */
+  .caButton {
+    /* Layout */
+    display: inline-block;
+    margin-right: 1rem;
```

Comment:
ideally you wouldn't set a margin on a button. but instead have a parent element around buttons that determines the spacing between them.

that said, i realize the webgui has a margin on buttons. Which in the near future I hope to change.

so don't feel like you need to change anything here.